src/Symfony/Component/Translation/Util/XliffUtils.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Util;
+
+use Symfony\Component\Config\Util\XmlUtils;
+use Symfony\Component\Translation\Exception\InvalidResourceException;
+
+/**
+ * XliffUtils is a bunch of utility methods to Xliff operations.
+ *
+ * This class contains static methods only and is not meant to be instantiated.
+ *
+ * @author Marcos D. Sánchez <marcosdsanchez@gmail.com>
+ */
+class XliffUtils
```

Comment:
Yes, but [XmlUtils](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Config/Util/XmlUtils.php#L22) that is a similar class is not final. I think that we should be consistent here.